### PR TITLE
VIH-6237 judge view participants panel

### PR DESCRIPTION
**Pull request by valentinabennett**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/VIH-6237

### Change description ###
Judge participants panel in hearing
**Does this PR introduce a breaking change?** (check one with "x")

```
[ X] Yes
[ ] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.html [hunk 1]
```diff
@@ -17,6 +17,10 @@
   </div>
 </div>
 
+<ng-container *ngIf="connected && showVideo">
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.html:20]: Please move this inside the video-container as discussed previously

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+              src="/assets/images/collapse.png"
+              class="panel-icon icon-op"
+              alt="Collapse panel icon"
+              id="collapse-icon-id"
+              (click)="toggleCollapseExpand()"
+            />
+          </span>
+        </span>
+        <ng-template #collapsePanel>
+          <span class="govuk-!-margin-bottom-3 vh-collaps">
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.html:17]: There's a typo in vh-collaps

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.scss [hunk 3]
```diff
@@ -0,0 +1,94 @@
+@import '~govuk-frontend/govuk/all';
+
+.panel-container {
+  background: govuk-colour('black');
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.scss:4]: We should avoid fixing the position of the component here, rather have the judge waiting room dictate where to place the participants panel

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.scss [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+.vh-name {
+  display: inline-block;
+  width: 210px;
+}
+
+.vh-twidth {
+  width: 100px;
+}
+
+.grey-btn {
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.scss:52]: Is govuk secondary button not suitable here?

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.ts [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+            }
+        });
+    }
+
+    toggleCollapseExpand() {
+        this.expandPanel = !this.expandPanel;
+    }
+
+    muteAll() {
+        this.isMuteAll = !this.isMuteAll;
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.ts:45]: Does this not need to call pexip to mute everyone?

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.ts [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+    }
+
+    lowerAllHands() {
+        this.isLowerAllHands = !this.isLowerAllHands;
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.ts:49]: Does this not need to call pexip to clearAllBuzz

Comment (valentinabennett) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.ts:49]: This functionality is not yet in this story, but I think it's button to lower all hands ( not toggle)

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.ts (hunk 5 above)
Comment (valentinabennett) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.ts:45]: Again it's will be in another story, but we will need functionality to mute all ( I don't know about unmute, toggle). I just test icon. I could remove it, as it only for test and will be replace in another story

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.html (hunk 1 above)
Comment (valentinabennett) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.html:20]: np, will move

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.html (hunk 2 above)
Comment (valentinabennett) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.html:17]: done

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.scss (hunk 4 above)
Comment (valentinabennett) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.scss:52]: no, Neil has it different, I  followed to his prototype

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.scss (hunk 3 above)
Comment (valentinabennett) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.scss:4]: I created css class to wrap panel on the host page and removed positioning from the panel.

**Pull request merged**